scripts/reboot

```diff
@@ -11,6 +11,8 @@ PLATFORM_UPDATE_REBOOT_CAUSE="platform_update_reboot_cause"
 REBOOT_CAUSE_FILE="/host/reboot-cause/reboot-cause.txt"
 PLATFORM_REBOOT_PRE_CHECK="platform_reboot_pre_check"
 REBOOT_TIME=$(date)
+PLATFORMS_NOT_SUPPORT_180S_WD=("x86_64-mlnx_msn2100-r0" "x86_64-mlnx_msn2010-r0")
```

Comment:
This deny-list approach is reactive, and not proactive. Is there a way the support can be detected automatically? If you can add such a support for MLNX only platforms then is that still better that this?